packages/core-svelte/src/components/TextDropdownComponent.svelte

```diff
@@ -336,6 +333,26 @@
         }
     };
 
+    const onFocusOut = () => {
```

Comment:
Please remove the methods that only log something, or are you not sure that you made the right choices?

Reply:
Removed